### PR TITLE
UI update

### LINKED ISSUE #91: Changes to UI
- [ ] Update the colours to be more aesthetically pleasing (look at past year examples + some UI guides)
- [ ] Change from using “Label” to a “Box?” idk what its called - but it was previously used by “Tag”
- [ ] Remove the delete entry button
- [ ] edit and note commands should show the intern application in the side panel

### PR DESCRIPTION
**Pull request by zhengyup**
Fixes #91 

Update the UI and edit Status Commands 

- change the UI from dark theme to light. Dark still available but need to edit some stuff to revert to that
- remove delete button 
- ensure edit and note commands select the changed application in the select view 
- add 'Interview', 'Offered', 'Assessment' statuses

### CODE REVIEW

File: src/main/resources/view/InternApplicationListPanel.fxml [hunk 1]
```diff
@@ -3,6 +3,6 @@
 <?import javafx.scene.control.ListView?>
 <?import javafx.scene.layout.VBox?>
 
-<VBox xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
-  <ListView fx:id="internApplicationListView" VBox.vgrow="ALWAYS" />
+<VBox fx:id="internApplicationVbox" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
```
Comment (tyouwei) [inline src/main/resources/view/InternApplicationListPanel.fxml:6]: I think you can remove the `fx:id="internApplicationVbox"` here, it doesn't have a reference anywhere in the code.

File: src/main/java/seedu/letsgethired/ui/InternApplicationCard.java [hunk 2]
```diff
@@ -14,6 +15,13 @@
 public class InternApplicationCard extends UiPart<Region> {
 
     private static final String FXML = "InternApplicationListCard.fxml";
+    private static final String ACCEPTED_COLOR = "-fx-text-fill: black; -fx-background-color: #7e38b7;";
+    private static final String ASSESSMENT_COLOR = "-fx-text-fill: black; -fx-background-color: #ffd100;";
+    private static final String INTERVIEW_COLOR = "-fx-text-fill: black; -fx-background-color: #fd5602;";
+    private static final String OFFERED_COLOR = "-fx-background-color: #03C04A;";
+    private static final String PANEL_COLOR = "-fx-background-color: #E9E9E9;";
+    private static final String PENDING_COLOR = "-fx-background-color: #209cee;";
+    private static final String REJECTED_COLOR = "-fx-background-color: #d30000;";
```
Comment (tyouwei) [inline src/main/java/seedu/letsgethired/ui/InternApplicationCard.java:24]: Great choices of colours! I think it'll be better if these properties are set in the css file. This is not urgent.

**Review by tyouwei — APPROVED** [review]
LGTM! There are some modifications that violate code consistencies, but due to time constraints for our CS2101 product demo, you can push this PR first. 

The comments I gave below are for myself to reference to in future. I will change them along with the StatusEnum in v1.4. 

Overall thanks for the good work in the UI!!!

File: src/main/java/seedu/letsgethired/ui/UiManager.java [hunk 3]
```diff
@@ -43,6 +43,7 @@ public void start(Stage primaryStage) {
             mainWindow = new MainWindow(primaryStage, logic);
             mainWindow.show(); //This should be called before creating other UI parts
             mainWindow.fillInnerParts();
+            primaryStage.setResizable(false);
```
[the remark this answers is not in the document]
Comment (zhengyup) [reply src/main/java/seedu/letsgethired/ui/UiManager.java:46]: Yep !

File: src/main/resources/view/StatusBarFooter.fxml [hunk 4]
```diff
@@ -4,7 +4,12 @@
...
 <?import javafx.scene.layout.GridPane?>
 
-<GridPane styleClass="status-bar" xmlns="http://javafx.com/javafx/17" xmlns:fx="http://javafx.com/fxml/1">
+<?import javafx.geometry.Insets?>
+<GridPane styleClass="pane-with-border" xmlns="http://javafx.com/javafx/17"
+          xmlns:fx="http://javafx.com/fxml/1">
+  <padding>
+    <Insets top="5" right="10" bottom="5" left="10" />
+  </padding>
   <columnConstraints>
```
[the remark this answers is not in the document]
Comment (zhengyup) [reply src/main/resources/view/StatusBarFooter.fxml:13]: It was looking abit weird and didnt fill the whole bottom so there was white space at the sides. The additions make it fill horizontally and give it some padding i believe, i was just adding here n there

File: src/main/java/seedu/letsgethired/ui/InternApplicationCard.java (hunk 2 above)
Comment (zhengyup) [reply src/main/java/seedu/letsgethired/ui/InternApplicationCard.java:24]: Thanks for the suggestion! I was only sure of how to conditionally set the colours in the Java file so i put it here. Same for the above odd/even issue

**Comment by zhengyup** [comment]
Thanks for the review @tyouwei ! There are also unused classes in the css file but not urgent also, can be done at the later date.

**Pull request merged**